### DISCUSSION TITLE
drive overview

### DISCUSSION THREAD

**Discussion by toohoo** [doublecmd/doublecmd#999] [General]
Hello,
I could not find the drive overview. I also searched in help files.
Please help, thanks in advance

**Comment by JonnyTech** [comment]
Please define "drive overview"? Is this a feature request?

**Reply by toohoo** [reply]
It is no feature request. I did suppose, the function is implemented already. 
In Total Commander one can select a drive for a panel with a drop down box. One item in the dropdown is named "computer (drive overview)". If this is selected the panel shows all available drives of the computer. And if the view for the panel is set to "long" (all details) then it is shown (free size)/(complete size).

**Reply by alexx2000** [reply]
DC does not have a such function.

**Reply by toohoo** [reply]
thanks

**Comment by toohoo** [comment]
function not implemented
